src/Core/Services/Implementations/SendService.cs

```diff
@@ -43,6 +51,25 @@ public class SendService : ISendService
 
         public async Task SaveSendAsync(Send send)
         {
+            // Make sure user can save Sends
+            if (send.UserId.HasValue)
+            {
```

Comment:
Should we also check to make sure user is part of an org first too to save cycles here? CurrentContext should have that.